base/strings/basic.jl

```diff
@@ -128,9 +128,7 @@ isless(a::AbstractString, b::AbstractString) = cmp(a,b) < 0
 cmp(a::String, b::String) = lexcmp(a.data, b.data)
 cmp(a::Symbol, b::Symbol) = Int(sign(ccall(:strcmp, Int32, (Cstring, Cstring), a, b)))
```

Comment:
I was thinking about `==` for `Symbol`, but actually it calls `===` so I guess that's OK already.